proto/ibc/applications/interchain_accounts/controller/v1/query.proto

```diff
@@ -5,16 +5,34 @@ package ibc.applications.interchain_accounts.controller.v1;
 option go_package = "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types";
 
 import "ibc/applications/interchain_accounts/controller/v1/controller.proto";
+import "gogoproto/gogo.proto";
 import "google/api/annotations.proto";
 
 // Query provides defines the gRPC querier service.
 service Query {
+  // InterchainAccount returns the interchain account address for a given owner address on a given connection
+  rpc InterchainAccount(QueryInterchainAccountRequest) returns (QueryInterchainAccountResponse) {
+    option (google.api.http).get =
+        "/ibc/apps/interchain_accounts/controller/v1/owner/{owner}/connection/{connection_id}";
```

Comment:
Should we use the more REST-ish style of plural nouns like we have in other routes?

```suggestion
        "/ibc/apps/interchain_accounts/controller/v1/owners/{owner}/connections/{connection_id}";
```